include/mbedtls/aes.h

```diff
@@ -112,8 +115,8 @@ void mbedtls_aes_free( mbedtls_aes_context *ctx );
  *                 <li>192 bits</li>
  *                 <li>256 bits</li></ul>
  *
- * \return         \c 0 on success or #MBEDTLS_ERR_AES_INVALID_KEY_LENGTH
- *                 on failure.
+ * \return         \c 0 on success.
+ *                 #MBEDTLS_ERR_AES_INVALID_KEY_LENGTH on failure.
```

Comment:
Other files use multiple `\return` statements.

Reply:
This file too.

Reply:
fixed